frontend/src/pages/MilestonesEditor.module.scss

```
@@ -0,0 +1,100 @@
+@import '../shared.scss';
+
+.horizontalScroller {
+  margin-left: 8px;
+}
+
+.addVersionButton {
+  width: 250px;
+  justify-content: center;
+  background-color: transparent;
+  cursor: pointer;
+  font-weight: bold;
+  font-size: 14px;
+  text-transform: uppercase;
+  svg {
+    margin: 4px;
+  }
+}
+
+.unassignedTasksCol {
+  flex: 1;
+  max-width: 350px;
+  min-width: 350px;
+  overflow-y: visible;
+  overflow-x: visible;
+  margin-bottom: 16px;
+  background-color: transparent;
+}
+
+.milestoneCol {
+  min-width: 350px;
+  max-width: 350px;
+  overflow-y: visible;
+  overflow-x: visible;
+  background-color: transparent;
+  border-radius: 8px;
+  margin-left: 16px;
+  margin-bottom: 16px;
+}
+
+.unassignedTasksHeader {
+  width: 100%;
+  padding-top: 8px;
+  padding-bottom: 8px;
+  font-size: 16px;
+  font-weight: bold;
+  border-bottom: 1px solid gray;
+}
+
+.milestoneHeader {
+  width: 100%;
+  border-bottom: 1px solid gray;
+  padding: 8px;
+  font-size: 16px;
+}
+
+.milestoneFooter {
+  display: flex;
+  justify-content: space-around;
+  width: 100%;
+  border-top: 1px solid gray;
+  padding: 8px;
+  font-size: 16px;
+}
+
+.sortableListWrapper {
+  overflow-y: auto;
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 8px;
+  min-height: 100px;
+  padding: 16px;
+  padding-top: 8px;
+  scrollbar-width: thin;
+}
+
+.milestoneWrapper {
+  display: flex;
```

```suggestion
.milestoneWrapper {
  @extend .shadow-roadmapblock;
  display: flex;
```

Drop shadow tyylit voisi kuitenkin sisällyttää näihin, niin ei tarvi aina käytettäessä muistaa lisätä kahta luokkaa komponentille. (sama `.graphItem` luokille)

Ja designissa näkyi käytetyn tuota "roadmapblock" shadow:ta

---

Kiitos extend-vinkkauksesta, en tiennytkään sen olemassaolosta.